### PR TITLE
Autocomplete

### PR DESCRIPTION
**Pull request by Kumjami**
As described in https://github.com/numo-labs/isearch-ui/issues/265#issuecomment-225859471

### CODE REVIEW

**Comment by Kumjami** [comment]
Update: 

### Current implementation

image

### Design implementation 

image

**Comment by nelsonic** [comment]
👀  (_reviewing after standup..._)

File: lib/search-bar/defaultSuggestions.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/**
+ * If there is no suggestions matches this list will be displayed
+ */
+export default [{
+  boldTitle: 'Oh no! ',
```
Comment (nelsonic) [inline lib/search-bar/defaultSuggestions.js]: We need these strings to be translated to Danish ... 
@emiledskar can you help?

File: lib/search-bar/index.js [hunk 2]
```diff
@@ -1,17 +1,53 @@
 import React, { Component, PropTypes } from 'react';
-var Typeahead = require('react-typeahead').Typeahead;
+import Autosuggest from 'react-autosuggest';
```
Comment (nelsonic) [inline lib/search-bar/index.js]: This looks like a _better-tested_ component. 👍

File: lib/search-bar/defaultSuggestions.js (hunk 1 above)
Comment (emiledskar) [inline lib/search-bar/defaultSuggestions.js]: I would go with "Desværre ingen resultater på din søgning..."
and "Måske kan du prøve følgende i stedet"
errormessage

**Comment by Kumjami** [comment]
@nelsonic @emiledskar translated :+1: 

image

**Comment by jackcarlisle** [comment]
looks good 👍

**Pull request merged**